tests/crud/test_step.py

```diff
@@ -0,0 +1,31 @@
+"""Tests of the CRUD for the step model."""
+from arnold.adapter import ArnoldAdapter
+from arnold.crud.create import create_step
+from arnold.crud.read.step import get_step
+from arnold.models.database import Step
+
+
+def test_create_step(valid_step: Step, mock_adapter: ArnoldAdapter):
+    """Test to create a step in the database."""
+    # GIVEN a valid step
+
+    # WHEN creating a step in the database
+    create_step(adapter=mock_adapter, step=valid_step)
+
+    # THEN a step can be retrieved from the database
+    step: dict = mock_adapter.step_collection.find_one({"prep_id": valid_step.prep_id})
+    assert step
+    assert isinstance(step, dict)
```

Comment:
Maybe an assertion about the content of the dict would be nice

Reply:
fixed